src/maps.rs

```diff
@@ -146,10 +185,17 @@ where
         line.clear();
     }
 
+    if !is_sorted_by_key(&entries, |entry| entry.range.start) {
```

Comment:
I believe this check is unnecessary overhead. The way that kernel generates is guarantees that it is ordered, and I don't see why that would ever change.